python/ray/data/_internal/execution/resource_manager.py

```diff
@@ -240,7 +230,7 @@ def __init__(self, resource_manager: ResourceManager):
         self._resource_manager = resource_manager
 
     @abstractmethod
-    def update_usages(self) -> ExecutionResources:
```

Comment:
Why're we changing the type hint? Is this method supposed to return `None`?

Reply:
yes, it was a mistake